Gemfile.lock

```diff
@@ -382,6 +391,3 @@ DEPENDENCIES
   uglifier (>= 1.0.3)
   valid_attribute
   validate_url
-
-BUNDLED WITH
-   1.10.6
```

Comment:
@gabe-k What version of Bundler are you using? This line was introduced to Bundler a little while ago, and I think only old versions erase it.